### PR TITLE
Mark `requests` stubs as complete

### PR DESCRIPTION
**Pull request by AlexWaygood**
Co-authored-by: kkirsche <kevin.kirsche@verizon.com>

### CODE REVIEW

File: stubs/requests/requests/sessions.pyi [hunk 1]
```diff
@@ -8,6 +8,7 @@ from urllib3._collections import RecentlyUsedContainer
 from . import adapters, auth as _auth, compat, cookies, exceptions, hooks, models, status_codes, utils
 from .models import Response
 from .structures import CaseInsensitiveDict as CaseInsensitiveDict
+from .utils import DEFAULT_PORTS as DEFAULT_PORTS
```
Comment (JelleZijlstra) [inline stubs/requests/requests/sessions.pyi:11]: Any particular reason to re-export this? Looks like at runtime it's a regular import.

File: stubs/requests/requests/sessions.pyi [hunk 2]
```diff
@@ -40,10 +41,23 @@ def merge_setting(request_setting, session_setting, dict_class=...): ...
...
 
 class SessionRedirectMixin:
-    def resolve_redirects(self, resp, req, stream=..., timeout=..., verify=..., cert=..., proxies=...): ...
+    def resolve_redirects(
+        self,
+        resp,
+        req,
+        stream: bool = ...,
+        timeout: Incomplete | None = ...,
+        verify: Incomplete | None = ...,
```
Comment (JelleZijlstra) [inline stubs/requests/requests/sessions.pyi:50]: I think this is just a bool, at runtime it's `verify=True`.

File: stubs/requests/requests/utils.pyi [hunk 3]
```diff
@@ -57,3 +62,7 @@ def to_native_string(string, encoding=...): ...
 def urldefragauth(url: _Uri): ...
 def rewind_body(prepared_request: PreparedRequest) -> None: ...
 def check_header_validity(header: tuple[AnyStr, AnyStr]) -> None: ...
+
+if sys.platform == "win32":
+    def proxy_bypass_registry(host) -> bool: ...
+    def proxy_bypass(host) -> bool: ...
```
Comment (JelleZijlstra) [inline stubs/requests/requests/utils.pyi:68]:
```suggestion
    def proxy_bypass_registry(host: str) -> bool: ...
    def proxy_bypass(host: str) -> bool: ...
```
The implementation does `'.' in host`.

File: stubs/requests/requests/sessions.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stubs/requests/requests/sessions.pyi:11]: Yeah, I was on the fence on whether to re-export this or just allowlist it. I'll allowlist it.

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**